charts/spark-operator-chart/templates/cert-manager-certificate.yaml

```diff
@@ -0,0 +1,28 @@
+{{- $webhookSvc := printf "%s-webhook.%s" (include "spark-operator.fullname" .) .Release.Namespace -}}
+{{- if and .Values.webhook.enable .Values.webhook.certManager.enable }}
+apiVersion: cert-manager.io/v1alpha3
+kind: Certificate
+metadata:
+  name: {{ include "spark-operator.fullname" . }}-webhook-cert
+  labels:
+    {{- include "spark-operator.labels" . | nindent 4 }}
+spec:
+  dnsNames:
+    - {{ $webhookSvc }}.svc
+    - {{ $webhookSvc }}.svc.cluster.local
+  duration: 87600h0m0s
```

Comment:
I don't know if it will be better to make this configurable and move the default value either to values.yaml or here with `default`